server/eclipse-project/src/main/java/us/freeandfair/corla/model/DoSDashboard.java

```diff
@@ -263,16 +263,6 @@ public boolean updateContestToAudit(final ContestToAudit the_contest_to_audit) {
       .map(a -> a.contest());
   }
 
-  /**
-   * data access helper
-   * @return contests of contestsToAudit a.k.a selected contests, targeted contests
-   */
-  // FIXME this might not be needed.
-  public Stream<String> targetedContestNames() {
-    return contestsToAudit().stream()
-      .map(a -> a.contest().name());
-  }
```

Comment:
Totally unused before, or removed a call site in this PR?

Reply:
Totally unused.